### ISSUE TITLE
input files

### ISSUE THREAD

**Issue by Thexiyang** [linxingchen/cobra.github.io#1]
Thanks for this nice tool! Is it possible to add advice in the readme text on how produce coverage.txt  and mapping.sam ?

**Comment by linxingchen** [comment]
Yes. thank you for your suggestion. I will add it soon.

**Comment by linxingchen** [comment]
I have added the descriptions in the readme text, please let me know if you need more information. Thank you.

**Comment by Thexiyang** [comment]
Is it a must that that contigs should be from exactly the same reads that the contigs are assembled from?

**Comment by linxingchen** [comment]
> Is it a must that that contigs should be from exactly the same reads that the contigs are assembled from?

Yes. You need that to have the correct coverage information and also contigs spanned by the paired reads. To clarify, the two paired read files should be the same as you used for assembly.

**Comment by Thexiyang** [comment]
Thanks
